package.json

```diff
@@ -113,7 +113,7 @@
     "cookie-parser": "^1.4.4",
     "core-decorators": "^0.20.0",
     "cors": "^2.8.5",
-    "crypto-js": "^3.1.9-1",
+    "crypto-js": "^4.1.1",
```

Comment:
crypto-js upgrade necessary for URL safe base64 encoder for sha256 toString method